datastore/tests/unit/test_entity.py

```diff
@@ -221,6 +222,18 @@ def test___repr___w_key_non_empty(self):
         expected = "<Entity%s %s>" % (flat_path, entity_vals)
         self.assertEqual(repr(entity), expected)
 
+    def test__set_item_enum(self):
+        """Test if given Enum converts to int."""
+        entity = self._make_one(key=_Key())
+        entity['some_value'] = _IntEnum.VALUE_1
+
+        self.assertEqual(type(entity['some_value']), int)
```

Comment:
Type check needed, 'cause enum and int are comparable, so second assert will be True no matter enum or int stored in Entity